js/components/user/list.vue

```diff
@@ -58,6 +58,13 @@ export default {
                 align: 'center',
                 type: 'metric',
                 width: 95
+            }, {
+                label: this._('Roles'),
+                key: 'roles',
+                sort: 'roles',
```

Comment:
Sorting on roles does not seems relevant to me. Which role is the sort key when many roles ?